climpred/tests/test_compute_dims.py

```diff
@@ -98,6 +98,7 @@ def test_bootstrap_pm_dim(PM_da_initialized_1d, PM_da_control_1d):
         dim='member',
         comparison='m2c',
         bootstrap=BOOTSTRAP,
+        resample_dim='member',
```

Comment:
Should we add a separate test or a parametrize here to try both member and init?

Reply:
yep. i will add this